normandy/schema.py

```diff
@@ -9,4 +9,16 @@ class NormandyQuery(BaseQuery, RecipesQuery, StudiesQuery, graphene.ObjectType):
     pass
 
 
-schema = graphene.Schema(query=NormandyQuery)
+class DisableIntrospectionMiddleware:
+    """
+    This class hides the introspection.
```

Comment:
Could you make it explicit why we are doing that? 
Link to bug? A few words maybe?

Reply:
since there's no bug, i added a link that states its best practice to disable introspection queries. Though, I could great a git issue and reference it?

Reply:
It's ok ;)